_beats/libbeat/scripts/Makefile

```diff
@@ -72,7 +72,7 @@ TESTING_ENVIRONMENT?=snapshot## @testing The name of the environment under test
 BEAT_VERSION=$(shell head -n 1 ${ES_BEATS}/libbeat/docs/version.asciidoc | cut -c 17- )
 COMMIT_ID=$(shell git rev-parse HEAD)
 DOCKER_COMPOSE_PROJECT_NAME?=${BEAT_NAME}${TESTING_ENVIRONMENT//-}${BEAT_VERSION//-}${COMMIT_ID} ## @testing The name of the docker-compose project used by the integration and system tests
-DOCKER_COMPOSE?=TESTING_ENVIRONMENT=${TESTING_ENVIRONMENT} ES_BEATS=${ES_BEATS} docker-compose -p ${DOCKER_COMPOSE_PROJECT_NAME} -f docker-compose.yml
+DOCKER_COMPOSE?=TESTING_ENVIRONMENT=${TESTING_ENVIRONMENT} ES_BEATS=${ES_BEATS} USER_ID="$(shell id -u):$(shell id -g)" docker-compose -p ${DOCKER_COMPOSE_PROJECT_NAME} -f docker-compose.yml
```

Comment:
IIRC you will lose those changes when you upgrade libbeat

Reply:
Yep, when I created it was only for validating this fix. If it works then I'll raise the PR in the upstream :)

Reply:
`make fix-permissions` resolves the issue I have added it at https://github.com/elastic/apm-server/pull/2948/files

Reply:
Awesome, thanks for the hint :)